source/Mlos.Python/setup.py

```diff
@@ -9,14 +9,14 @@
     version="0.0.1",
     packages=find_packages(),
     install_requires=[
-        'bayesian-optimization==1.0.1',
-        'scikit-learn==0.22.1',
-        'scipy==1.3.2',
-        'numpy==1.18.1',
-        'pandas==1.0.3',
-        'pylint==2.3.1',
-        'pyodbc==4.0.30',

```

Comment:
I don't like this but I can't get pip to pick up on the conda version of pyodbc. conda shows 4.0.30, pip shows 4.0.0-unsupported.

Reply:
But they're exact version matches in the environment.yml?  Shouldn't they be consistent?

Reply:
When they match, it doesn't work, is what I'm saying.

Reply:
Weird, but okay.

Reply:
yes, if you conda install 4.0.30, conda list shows 4.0.30, but pip shows 4.0.0-unsupported which is not >=4.0 and not even >3. So I didn't really know what else to do. Going to pip for pyodbc doesn't seem to be an option because many wheels seem to be missing.